app/views/bgeigie_imports/_edit.html.erb

```diff
@@ -98,7 +98,8 @@
       %>
       <%= f.input :subtype,
                   :collection => %w(Drive Surface Cosmic None),
-                  :include_blank => false,
+                  :selected => "Drive",
+		  :include_blank => false,
```

Comment:
```suggestion
                  :include_blank => false,
```

You know github has a button for that now @eitoball ?

Reply:
Ah, yes.